app/javascript/components/challenges/Cards.tsx

```diff
@@ -1,8 +1,47 @@
 import * as React from 'react';
+import Carousel from "./Carousel"
 
-export default function Cards() {
-  const nasaApiKey = '6H6EdNLLrDu8SC1LZMJkbJzoGIghjvrjzgQpF72W';
-  const baseUri = 'https://api.nasa.gov/planetary/apod';
+const nasaApiKey = '6H6EdNLLrDu8SC1LZMJkbJzoGIghjvrjzgQpF72W';
+const baseUri = 'https://api.nasa.gov/planetary/apod';
+// not the most elegant date randomiser, I know, but works (sort of)
+var myDate = "2019-0"+Math.floor(Math.random()*12).toString() +"-" +  Math.floor(Math.random()*29).toString();
+
+
+function getImage(date: string) {
+  return fetch(`${baseUri}?api_key=${nasaApiKey}&date=${myDate}`)
```

Comment:
You don't need to use myDate here since we are passing the date into the method.

Just needs to be `return fetch('${baseUri}?api_key=${nasaApiKey}&date=${date}')`